index.html

```diff
@@ -914,6 +827,58 @@ <h3>
         </p>
       </section>
     </section>
+    <section>
+      <h2>
+        Dependencies
+      </h2>
+      <p>
+        The following concepts and interfaces are defined in [[HTML]]:
+        <dfn data-cite="HTML/webappapis.html#event-handlers">event
+        handler</dfn>, <dfn data-cite=
+        "HTML/webappapis.html#event-handler-event-type">event handler event
+        type</dfn>, <dfn data-cite=
+        "HTML/webappapis.html#concept-task">task</dfn>, <code><dfn data-cite=
+        "HTML/window-object.html#window">Window</dfn></code>,
+        <code><dfn data-cite="HTML/dom.html#document">Document</dfn></code>,
+        <dfn data-cite="HTML/browsers.html#browsing-context">browsing
+        context</dfn>, <dfn data-cite=
+        "HTML/browsers.html#top-level-browsing-context">top-level browsing
+        context</dfn>, browsing context's <dfn data-cite=
+        "HTML/browsers.html#active-document">active document</dfn>,
+        <dfn data-cite="HTML/browsers.html#navigate">navigated</dfn> browsing
+        context, <dfn data-cite=
+        "HTML/browsers.html#active-sandboxing-flag-set">active sandboxing flag
+        set</dfn>, <dfn data-cite=
+        "HTML/browsers.html#sandboxed-orientation-lock-browsing-context-flag">sandboxed
+        orientation lock browsing context flag</dfn>, <dfn data-cite=
+        "HTML/webappapis.html#responsible-document">responsible
+        document</dfn>, <dfn data-cite=
+        "HTML/browsers.html#list-of-the-descendant-browsing-contexts">list of
+        the descendant browsing contexts</dfn>.
+      </p>
+      <p>
+        The following is defined in [[FULLSCREEN]]: <dfn data-cite=
+        "FULLSCREEN/#fullscreen-element">fullscreen element</dfn>.
+      </p>
+      <p>
+        The following is defined in [[PAGE-VISIBILITY]]: <dfn data-cite=
+        "PAGE-VISIBILITY/#dfn-now-visible-algorithm">now visible
```

Comment:
Just noticed a tiny thing that was a mistake from before, the "/" below was not needed. 

```suggestion
        "PAGE-VISIBILITY#dfn-now-visible-algorithm">now visible
```